packages/manager/modules/dbaas-logs/src/logs/detail/tokens/add/logs-token-add.html

```diff
@@ -23,8 +23,8 @@
         <oui-field
             data-label="{{ ::'logs_tokens_select_cluster' | translate }}"
         >
-            <oui-spinner data-ng-if="ctrl.clusters.loading"></oui-spinner>
-            <div class="oui-select" data-ng-if="!ctrl.clusters.loading">
+            <oui-spinner data-ng-show="ctrl.clusters.loading"></oui-spinner>
+            <div class="oui-select" data-ng-hide="ctrl.clusters.loading">
```

Comment:
**question:** May I know why `ng-if ` has been changed to `ng-show` and `ng-hide`? We should consider using `ng-if` since it removes the element from DOM and not have hidden elements in the DOM unless there is a particular reason to let the element stay in the DOM.

Reply:
Thanks for the explanation, i will do as you say.